### ISSUE TITLE
Forms filled out on a PDF sheet don't persist between open/close

### ISSUE THREAD

**Issue by sunbeam60** [Heilemann/Realms#15]
https://github.com/user-attachments/assets/c8837fb9-5778-4763-99a2-b9c1d55f9380

**Comment by Heilemann** [comment]
Do me a favor and try that again. I was making some changes in this area just an hour ago that should have deployed.

**Comment by sunbeam60** [comment]
The bug is super fresh and it still persists, unfortunately. Happens in both Firefox and Edge

**Comment by Heilemann** [comment]
I can only recreate this on the Knave sheet, not on other fillable sheets; is that also the case for you?

**Comment by sunbeam60** [comment]
Lmc later tonight

**Comment by sunbeam60** [comment]
Yes, confirmed works well with another character sheet. Also made me realise that every sheet creates a new category of item, which is bloody marvelous!!!
